paperlessIntegration/src/main/java/at/fhtw/swen3/paperless/integration/services/HealthChecker.java

```diff
@@ -0,0 +1,46 @@
+package at.fhtw.swen3.paperless.integration.services;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Getter
+@NoArgsConstructor
+@Component
+public class HealthChecker {
+    @Value("${healthcheck.server}")
+    private String serverAddress;
+
+    @Value("${healthcheck.port}")
+    private String port;
+
+    @Value("${healthcheck.endpoint}")
+    private String endpoint;
+
+    public boolean isHealthy() {
```

Comment:
At the moment is not used, if not needed can be removed

Reply:
:+1: It's now the only public method in that class, so we should keep it. ;)